src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/AuditUtility.cs

```diff
@@ -51,6 +51,9 @@ internal class AuditUtility
         internal int DistinctAdvisoriesSuppressedCount { get; private set; }
         internal int TotalWarningsSuppressedCount { get; private set; }
 
+        /// <inheritdoc cref="RestoreSummary.AuditRan"/>
+        internal bool AuditRan { get; private set; }
```

Comment:
I would just have `CheckPackageVulnerabilitiesAsync()` return `Task<bool>` and then save the value [here](https://github.com/NuGet/NuGet.Client/pull/6049/files#diff-fdeafb08a0d0ea6b92176fbc5d11e5eb726f77500e857cca8d8a646ff17fd379R536) to be returned.  Just my vote

Reply:
I agree, it's pretty much done that way anyways in the 3 places we return.